src/main/java/ro/unibuc/hello/controller/TaskController.java

```diff
@@ -74,4 +78,26 @@ public TaskDto getById (@PathVariable String id){
 		}
 	}
 
+	@PostMapping("/asign/{id}")
```

Comment:
That's not quite REST compliant. When writing RESTful APIs, the URL should not contain `method` names. The paths should describe resources and HTTP verbs used to describe the action. In this case, you will probably have to use a route like: `PATCH task/{id}/assignment`. 

More info about restful apis:
https://restfulapi.net/
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
https://stackoverflow.com/questions/28459418/use-of-put-vs-patch-methods-in-rest-api-real-life-scenarios